app.py

```diff
@@ -65,5 +75,76 @@ def delete(self):
             print("Error occured:", str(e.args))
             return Response('{"message":"Server error. Please check logs."}', status=400, mimetype='application/json')
 
+
+@ns_api_v1.route('/customer')
+class Customer(Resource):
+    global customerTable
+   # @marshal_with(store_marshal)
+
+    def get(self):
+        try:
+            cursor = customerTable.find({}, {"password" : 0})   # operation on table to get all data
+            response = []
+            for document in cursor: # iterate through each db result and append to a list
+                response.append(document)
+                return Response('{"response":%s,"message":"Succesfully retreived all documents"}' % dumps(response), status=200, mimetype='application/json')
+
+
+        except Exception as e:
+            print("Error occured:", str(e.args))
+            return Response('{"message":"Server error. Please check logs."}', status=400, mimetype='application/json')            
+
+
+    @ns_api_v1.expect(createCustomerSchema(api))
+    def post(self):
+        try:
+            customerId = getLastCustomerId(customerTable)
+            
+            if customerId == sys.maxsize:
+                raise Exception()
+            else:
+                customerId += 1
```

Comment:
This logic needs to be changed. So in mongo db a  unique ObjectID is created whenever you create a new document and hence you need not bother to add a logic to create unique ids.
Secondly this current logic of creating id is very dangerous in db security perspective. This leads to predicting ids of future users very easily. I know the scope of project is small and we can do this incrementing natural number until you reach the highest value but mongodb is already creating an id for you. You can see the oid created for the user document below

{
    "_id": {
        "$oid": "5eec747fa9d5841a633f97c9"
    },
    "id": 1,
    "name": "tanay shah",
    "phone": "6304632809",
    "username": "tanshah",
    "password": "$pbkdf2-sha256$29000$Xwsh5DyH0DpnrDXmnNM6xw$3Cp4yrMD8rO9xd9vJKcp7Q9xMz9a/yi6e/LyF45djZg"
}